timesketch/api/v1/resources/scenarios.py

```diff
@@ -269,3 +270,96 @@ def post(self, sketch_id, scenario_id):
             db_session.commit()
 
         return self.to_json(scenario)
+
+
+class QuestionConclusionListResource(resources.ResourceMixin, Resource):
+    """Resource for investigative question conclusion."""
+
+    @login_required
+    def get(self, sketch_id, question_id):
+        """Handles GET request to the resource.
+
+        Returns:
+            A list of JSON representations of the conclusions.
+        """
+        sketch = Sketch.query.get_with_acl(sketch_id)
+        question = InvestigativeQuestion.query.get(question_id)
+
+        conclusions = InvestigativeQuestionConclusion.filter_by(
+            investigativequestion=question
+        ).all()
+
+        if not sketch:
+            abort(HTTP_STATUS_CODE_NOT_FOUND, "No sketch found with this ID")
+
+        return self.to_json(conclusions)
+
+    @login_required
+    def post(self, sketch_id, question_id):
```

Comment:
This post function is placed in the class `QuestionConclusionList`. But it actually just modifies / adds one conclusion of a specific question. Wouldn't it make more sense to place this function in the `QuestionConclusionResource` below? With the **List** part of the class I wouldn't expect it to modify one conclusion to be honest.

Reply:
In REST and in our implementation, creating a new resource goes into the "list" part. For example, to create a new conclusion you would post to: `/sketches/1/questions/1/conclusions/` and to change an already existing one you PUT to `/sketches/1/questions/1/conclusions/123/`. I opted to not replicate the code, but I agree, you have a good point in this case. I'll add a PUT method on the QuestionConclusion for updates to make it more clear here.

Reply:
TIL, thanks!